packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingRadios.js

```diff
@@ -46,11 +46,15 @@ const ShippingRadios = props => {
         return { label, value };
     });
 
+    // Match the serialization of method objects from the useShippingMethod talon.
+    const { carrier_code, method_code } = selectedShippingMethod;
+    const selectedShippingMethodSerializedValue = `${carrier_code}|${method_code}`;
```

Comment:
This logic should probably move to `useShippingRadios` and then be handled by some utility function that is shared between `useShippingMethod` and `useShippingRadios` talons.

Reply:
👍 I had the same thought and was on the fence about it. 

Fixed in https://github.com/magento/pwa-studio/commit/7318d2bd0602c7c5bbbacad19747c57247c7b0fa.